Assets/FishFactory/Scripts/stunFish.cs

```diff
@@ -0,0 +1,26 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class stunFish : MonoBehaviour
+{
+    // counter for the number of fish that are stunned
+    public int nrFishStun = 0;
+    private void OnTriggerEnter(Collider collisionObject)
+    {
+        if (collisionObject.tag != "Destroyable") { return; }
```

Comment:
Should this be "Fish"?
Consistency with the despawner

Reply:
It only worked when the tag was set to "Destroyable", may be some inconsistencies in the fish prefab, not sure.